src/classes/com/sun/tdk/jcov/Instr.java

```diff
@@ -91,6 +97,8 @@ public class Instr extends JCovCMDTool {
         logger = Logger.getLogger(Instr.class.getName());
     }
 
+    boolean fixJavaBase = false;
+
```

Comment:
a field intended for interclass exchange (Instr<->JREInstr) is much better to draw up using g/setter && private access.